### DISCUSSION TITLE
UI feedback: Avoid automatic scroll-to-top behaviour when adding category assigments to elements

### DISCUSSION THREAD

**Discussion by mlopatka** [community/community#5915] [Projects and Issues]
When in the "By Category" sorted table view performing category assignment to issues that are currently classified as "No Category" upon selection of a category the viewport jumps to the top of the page due to the row being acted upon directly being sorted by its newly-assigned category. 

This is quite jarring and disrupts batch processing workflows when trying to categorize all uncategorized issues as a batch.

**Comment by paced** [comment]
+1. I'll also add: this prevents a very common Scrum procedure for backlog grooming from going efficiently.
